### PR TITLE
Services - last updated timestamp by day

### PR DESCRIPTION
**Pull request by aurelbojan**

### CODE REVIEW

File: src/main/java/com/resource/management/services/controller/TransferServicesController.java [hunk 1]
```diff
@@ -39,9 +39,14 @@ public ServicesListUpdatedNotification handle( final TransferServicesRequest req
...
-      final LastUpdatedTimestamp lastUpdatedTimestamp =
-            new LastUpdatedTimestamp( "timeStamp", Instant.now().toString() );
-      this.timestampRepository.save( lastUpdatedTimestamp );
-      return new ServicesListUpdatedNotification( this.repository.findAll(), lastUpdatedTimestamp.getTimeStamp() );
+
+      final LastUpdatedTimestamp timestampToday =
+            new LastUpdatedTimestamp( "timeStampToday", Instant.now().toString() );
+      final LastUpdatedTimestamp timestampTomorrow =
+            this.timestampRepository.findById( "timeStampTomorrow" ).orElse( null );
+      this.timestampRepository.save( timestampToday );
```
Comment (thejavatrainer) [inline src/main/java/com/resource/management/services/controller/TransferServicesController.java]: consider creating a rich timestampRepository interface

File: src/main/java/com/resource/management/services/controller/LastUpdatedTimestampService.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      }
+      else if ( "TOMORROW".equals( day ) )
+      {
+         timestampToday = this.timestampRepository.findById( "timeStampToday" ).orElse( null );
+         timestampTomorrow = new LastUpdatedTimestamp( "timeStampTomorrow", Instant.now().toString() );
+         this.timestampRepository.save( timestampTomorrow );
+      }
+      else
+      {
+         timestampToday = this.timestampRepository.findById( "timeStampToday" ).orElse( null );
```
Comment (thejavatrainer) [inline src/main/java/com/resource/management/services/controller/LastUpdatedTimestampService.java]: Use constants for the IDs and hide the find in a method in the repo, including the oreElse part.
The getTodaysTimestamp could return a nullable .
then the code would look cleaner.

Also, in my opinion a switch would look better.

File: src/main/java/com/resource/management/services/controller/DeleteServiceController.java [hunk 3]
```diff
@@ -10,27 +9,33 @@
...
+      String day = null;
+      if ( service.isPresent() )
+      {
+         day = service.get().getDay();
+      }
+
+      this.repository.deleteById( request.getId() );
+
+      System.out.println( "------------------DEBUGFGFGDFGFDGFDGFDGFD:--------- " + day );
+      return this.lastUpdatedTimestampService.getLastUpdatedNotification( day );
```
[the remark this answers is not in the document]
Comment (aurelbojan) [reply src/main/java/com/resource/management/services/controller/DeleteServiceController.java]: Not clear which method call might cause NPE.

[the remark this answers is not in the document]
Comment (thejavatrainer) [reply src/main/java/com/resource/management/services/controller/DeleteServiceController.java]: this.lastUpdatedTimestampService.getLastUpdatedNotification( day );
day can be null.

**Pull request merged**